lnrpc/routerrpc/router_server.go

```diff
@@ -455,36 +455,95 @@ func (s *Server) GetMissionControlConfig(ctx context.Context,
 	error) {
 
 	cfg := s.cfg.RouterBackend.MissionControl.GetConfig()
-	return &GetMissionControlConfigResponse{
+
+	resp := &GetMissionControlConfigResponse{
 		Config: &MissionControlConfig{
-			HalfLifeSeconds:             uint64(cfg.PenaltyHalfLife.Seconds()),
-			HopProbability:              float32(cfg.AprioriHopProbability),
-			Weight:                      float32(cfg.AprioriWeight),
-			MaximumPaymentResults:       uint32(cfg.MaxMcHistory),
-			MinimumFailureRelaxInterval: uint64(cfg.MinFailureRelaxInterval.Seconds()),
+			MaximumPaymentResults: uint32(cfg.MaxMcHistory),
+			MinimumFailureRelaxInterval: uint64(
+				cfg.MinFailureRelaxInterval.Seconds(),
+			),
 		},
-	}, nil
+	}
+
+	switch cfg.EstimatorType {
+	case routing.AprioriEstimatorName:
+		resp.Config.Model = MissionControlConfig_APRIORI
+
+	case routing.BimodalEstimatorName:
+		resp.Config.Model = MissionControlConfig_BIMODAL
+
+	default:
+		return nil, fmt.Errorf("unknown estimator config type %v",
+			cfg.EstimatorType)
+	}
+
+	eCfg := cfg.EstimatorConfig
+
+	// Apriori parameters.
+	resp.Config.HalfLifeSeconds = uint64(eCfg.PenaltyHalfLife.Seconds())
```

Comment:
Do we need to set these still for both versions? Also what about the bimodal specific items?